src/main/java/org/kainos/ea/api/JobService.java

```diff
@@ -0,0 +1,25 @@
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.Job;
+import org.kainos.ea.db.JobDao;
+import org.kainos.ea.exception.FailedToGetJobsException;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class JobService {
+
+    private final JobDao jobDao = new JobDao();
```

Comment:
you should pass this in via constructor so you can mock it out in unit tests

Reply:
Updated to include this change